services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/AlluDataFactory.kt

```diff
@@ -336,36 +335,16 @@ class AlluDataFactory(
                 description = description,
             )
 
-        val hakijaApplicationContact =
-            ApplicationUserContact(
-                "Henri Hakija",
-                "henri.hakija@mail.com",
-                ApplicationContactType.HAKIJA
-            )
+        val hakijaContact = UserContact("Henri Hakija", "henri.hakija@mail.com")
```

Comment:
These values could be moved somewhere else, as they have nothing to do with Allu data. This factory class has grown bloated over time because the application stuff wasn't separated from this when we separated Haitaton application classes from Allu application classes. But these are also not Application data, rather something created from both applications and hanke. Maybe a separate `UserContactFactory` object? Though these are not factory methods, either.